docs/reference-architecture/video-transcoding/diagrams/figure3.svg

```diff
@@ -0,0 +1,6173 @@
+    <linearGradient
+       id="linear-gradient"
+       x1="1075.9991"
+       y1="-86.0011"
+       x2="3198.3987"
+       y2="2036.3986"
+       gradientUnits="userSpaceOnUse">
+      <stop
+         offset="0"
+         stop-color="#fd8e3a"
+         stop-opacity=".3"
+         id="stop2563" />
+      <stop
+         offset="1"
+         stop-color="#fd8e3a"
+         stop-opacity=".3"
+         id="stop2565" />
+    </linearGradient>
+    <linearGradient
+       id="linear-gradient-2"
```

Comment:
Looks like there are actually a bunch of these defined... they all should ideally be unique on the page... `id="svg3--linear-gradient-2"`